framework_tvb/tvb/adapters/exporters/export_manager.py

```diff
@@ -58,8 +58,6 @@ class ExportManager(object):
     all_exporters = {}  # Dictionary containing all available exporters
     export_folder = None
```

Comment:
This should have been moved to StorageInterface, and all usages throughout ABCExporter, ExportManager and actual exporters could be removed.